jupyterhub/utils.py

```diff
@@ -171,7 +171,15 @@ async def exponential_backoff(
         # add some random jitter to improve performance
         # this prevents overloading any single tornado loop iteration with
         # too many things
-        dt = min(max_wait, remaining, random.uniform(0, start_wait * scale))
+        # Except Overflow Error. Otherwise start_wait * scale will be too big at some point.
```

Comment:
Ignore my previous comment. Could this be simplified to something like:
```py
if dt < max_wait:
    dt = min(max_wait, remaining, random.uniform(0, start_wait * scale))
    scale *= scale_factor
```
else leave`dt` unchanged?

Reply:
Thanks for your response. I think any code snippet that prevents `scale *= scale_factor` from growing at some point will be fine.